### DISCUSSION TITLE
[Question]: local ollama nomic-embed-text does not work in Assistant

### DISCUSSION THREAD

**Discussion by barisgundem** [danny-avila/LibreChat#4703] [Troubleshooting]
### What is your question?

I setup nomic-embed-text ollama locally, in librechat I can upload files and make queries while chatting with openai, it works fine, but it does not work in the assistant section, although it says upload success in the console, it does not answer the query correctly.

Need help to integrate my local embedding and vectordb setup to work with Assistant.

### More Details

I shared 2 screenshots for better understanding.

Image from chat(which is successfully can query pdfs):
<img width="337" alt="Screenshot 2024-11-12 at 22 26 21" src="https://github.com/user-attachments/assets/fc510777-80f9-4b84-a51d-bbb951d2a6f7">

Image from assistant(ı can upload files but not query them)
<img width="340" alt="Screenshot 2024-11-12 at 22 27 48" src="https://github.com/user-attachments/assets/619bb06c-cac6-4609-8f9d-1194b6815322">

### What is the main subject of your question?

Installation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Assistants do not use the RAG API, they use OpenAI's file retrieval system, see: https://platform.openai.com/docs/assistants/overview

**Reply by barisgundem** [reply]
Thank you very much for your answer, I want to make a local rag system to use on the assistant side, is there any advice you can give me on this? Things I should pay attention to in the project etc. or tech stack recommendation for rag.
